packages/apps-config/src/api/spec/node-template.ts

```diff
@@ -7,5 +7,7 @@
 
 export default {
   Address: 'AccountId',
-  LookupSource: 'AccountId'
+  LookupSource: 'AccountId',
+  // Remove when https://github.com/substrate-developer-hub/substrate-node-template/ moves to alpha-v7
```

Comment:
I have a tentative plan to start maintaining these types on a per-node-template-version basis. That way older versions of the node template will still be usable after newer versions are released. Concretely, node template alpha.6 will still be usable after alpha.7 is released and neither will require custom type input.

Have to talk it over with devhub team still.